rxnutils/chem/utils.py

```diff
@@ -540,3 +583,73 @@ def split_smiles_from_reaction(smiles: str) -> List[str]:
         else:
             components.append(smiles[block_start:pos])
     return components
+
+
+def reaction_centres(rxn) -> Tuple[List[int], ...]:
```

Comment:
Missing type hint for `rxn`.

```suggestion
def reaction_centres(rxn: AllChem.ChemicalReaction) -> Tuple[List[int], ...]:
```